odoo/api.py

```diff
@@ -305,6 +306,28 @@ def method(self, args):
     return method
 
 
+def multi(method):
+    """ Decorate a record-style method where ``self`` is a recordset. The method
+        typically defines an operation on records. Such a method::
+
+            def method(self, args):
+                ...
+
+        may be called in both record and traditional styles, like::
+
+            # recs = model.browse(cr, uid, ids, context)
+            recs.method(args)
+
+            model.method(cr, uid, ids, args, context=context)
+
+        .. deprecated:: 13.0
+
+            multi is the default scheme, in 13.0+ this decorator is effectively a no-op
+    """
+    warnings.warn("@api.multi is deprecated since it's now the default scheme, please remove it")
```

Comment:
Maybe generate a `DeprecationWarning` for this one?
```suggestion
    warnings.warn("@api.multi is deprecated since it is now the default, please remove it",
                  DeprecationWarning)
```